app/preprints/-components/submit/review/template.hbs

```diff
@@ -1,67 +1,117 @@
+            </div>
+            <hr>
         </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.preprint-title'}}
+        <div local-class='step-container'>
+            <h3 local-class='title'
+                data-test-title-step-two
+            >
+                {{t 'preprints.submit.step-two.title'}}
+            </h3>
+            <div local-class='content-container'>
+                <div local-class='title'>
+                    {{t 'preprints.submit.step-five.contributors'}}
+                </div>
+                <div local-class='display'>
+                    <ContributorList
+                        @model={{this.preprint}}
+                        @shouldTruncate={{false}}
+                        @shouldLinkUsers={{true}}
+                    />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.title}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintLicense @preprint={{this.preprint}} />
+                </div>
             </div>
-        </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.description'}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintDiscipline @subjects={{this.subjects}} />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.description}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintTag @preprint={{this.preprint}} />
+                </div>
             </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-doi'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.preprintDoiUrl}}
+                </div>
+            </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-year'}}
+                </h4>
+                <div local-class='display'>
+                    {{moment-format this.preprint.originalPublicationDate 'YYYY-MM-DD'}}
+                </div>
+            </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-citation'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.publicationCitation}}
+                </div>
+            </div>
```

Comment:
Possible question for product team, but if these are optional, do we want to wrap `div .content-container` in a `{{#if this.preprint.preprintDoiUrl}}` and not show these fields? Alternatively we could always show these fields and just show a default "No publication citation" message.